lib/web/apiserver_test.go

```diff
@@ -5687,13 +5586,12 @@ func login(t *testing.T, clt *client.WebClient, cookieToken, reqToken string, re
 }
 
 func validateTerminalStream(t *testing.T, conn *websocket.Conn) {
+	t.Helper()
 	termHandler := newTerminalHandler()
 	stream := termHandler.asTerminalStream(conn)
-	_, err := io.WriteString(stream, "echo foo\r\n")
-	require.NoError(t, err)
-
-	err = waitForOutput(stream, "foo")
+	_, err := io.WriteString(stream, "echo txlxport | sed 's/x/e/g'\r\n")
```

Comment:
These tests were flaky for 2 reasons:

1. They try to start an SSH session as OS-user "user", which doesn't exist. Fix this by using the actual user with `user.Current()`.
2. They were running `echo foo` and searching for `foo`, but the `echo foo` on standard in is also in the websocket stream, so the test would pass even if we didn't receive the "foo" output from the echo command. Fix this by using a command that doesn't contain the output we're actually looking for.